pkg/sql/parser/sql.y

```diff
@@ -2772,6 +2774,24 @@ create_schedule_for_backup_stmt:
       ScheduleOptions:  $12.kvOptions(),
     }
   }
+
+|
+CREATE SCHEDULE /*$3=*/opt_description IF NOT EXISTS FOR BACKUP /*$6=*/opt_backup_targets INTO
+  /*$8=*/string_or_placeholder_opt_list /*$9=*/opt_with_backup_options
+  /*$10=*/cron_expr /*$11=*/opt_full_backup_clause /*$12=*/opt_with_schedule_options
+  {
```

Comment:
I wonder: would it make sense to introduce a non-terminal rule; something like:

```
schedule_label_spec:
  string_or_placeholder
  {
      $$.val = tree.ScheduleLabel{Label: $1.expr()}
  }
| IF NOT EXISTS string_or_placeholder
  {
      $$.val = tree.ScheduleLabel{Label: $4.expr, RequireUnique: true}
  }
| /* EMPTY */
  {
      $$.val = tree.ScheduleLabel{}
  }
```

We would of course have to add ScheduleLabel definition, and provide way to access it (scheduleLabel() method added in this file -- similar to e.g. `fullBackupClause`)

I think what this will enable you to have is just 1 clause for create schedule:
```
create_schedule_for_backup_stmt:
  CREATE SCHEDULE schedule_label FOR ...
```

Reply:
done